src/alire/alire.ads

```diff
@@ -9,7 +9,7 @@ with Simple_Logging;
 
 package Alire with Preelaborate is
 
-   Version : constant String := "0.7.0-dev";
+   Version : constant String := "0.7.1-dev";
```

Comment:
I think it should be `0.8.0` (see #564).
If some patches are interesting for a the `0.7.1` release we can cherry-pick them in the `release/0.7-beta` branch.

Reply:
About #564, is still in draft on purpose?

Reply:
I was waiting because we were still merging stuff into `0.7`.